testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeCompatibility.java

```diff
@@ -59,14 +69,22 @@ public void testInformationSchemaColumnsOnPresenceOfHiveView()
     @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
     public void testUnregisterNotDeltaLakeTable()
     {
+        String schemaName = "test_unregister_not_delta_lake_schema_" + randomNameSuffix();
+
         String baseTableName = "test_unregister_not_delta_table_" + randomNameSuffix();
-        String hiveTableName = "hive.default." + baseTableName;
+        String hiveTableName = format("hive.%s.%s", schemaName, baseTableName);
 
-        onTrino().executeQuery("CREATE TABLE " + hiveTableName + " AS SELECT 1 a");
+        try {
+            onTrino().executeQuery("CREATE SCHEMA IF NOT EXISTS hive." + schemaName + " with (location = 's3://" + bucketName + "/test_unregister_to_delta')");
 
-        assertThatThrownBy(() -> onTrino().executeQuery("CALL delta.system.unregister_table('default', '" + baseTableName + "')"))
-                .hasMessageContaining("not a Delta Lake table");
+            onTrino().executeQuery("CREATE TABLE " + hiveTableName + " AS SELECT 1 a");
 
-        onTrino().executeQuery("DROP TABLE " + hiveTableName);
+            assertThatThrownBy(() -> onTrino().executeQuery(format("CALL delta.system.unregister_table('%s', '%s')", schemaName, baseTableName)))
+                    .hasMessageContaining("not a Delta Lake table");
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE IF EXISTS " + hiveTableName);
+            onTrino().executeQuery("DROP SCHEMA " + schemaName);
```

Comment:
This line may throw an exception if `CREATE SCHEMA` failed. How about adding `IF EXISTS` option or moving `CREATE SCHEMA` before `try` block?